libcontainer/specconv/spec_linux.go

```diff
@@ -639,6 +639,7 @@ func parseMountOptions(options []string) (int, []int, string, int) {
 		clear bool
 		flag  int
 	}{
+		"acl":           {false, unix.MS_POSIXACL},
```

Comment:
This implementation might not be correct:
- https://github.com/opencontainers/runc/issues/3738